app/views/communicart_mailer/cart_notification_email.html.erb

```diff
@@ -31,7 +31,9 @@
     <tr id="approval-actions">
       <td colspan=2 align="center">
         <p>
-          <%= link_to "Approve all items with one click", approve_all_url(@approval), class: 'form-button' %>
+          <%= link_to "Approve all items with one click", approve_all_url(@approval, true), class: 'form-button' %>
+          <%= link_to "Reject all items with one click", approve_all_url(@approval, false), class: 'form-button' %>
```

Comment:
I would be in favor of these two updates:
* de-emphasizing the rejection action a bit by having a normal link. The easy way to do this may be to just remove the class, so that it isn't a button. Sorry, I did add the comment to make this a link in story but it mentions a button in the description.

* Just updating the text to say 'Reject all items', again to simplify it.

Outside of these two comments, :shipit: 
Thanks @sashax!